dffml/source/file.py

```diff
@@ -19,6 +19,7 @@
 
 class FileSourceConfig(BaseConfig, NamedTuple):
     filename: str
+    label: str
```

Comment:
Give this a default value of `"unlabeled"`

Reply:
done :+1: